.circleci/config.yml

```diff
@@ -216,6 +216,26 @@ jobs:
           name: Run danger-swift
           command: danger-swift ci
 
+  "Run Pod spec lint":
+    macos:
+      xcode: "11.2.0"
+    environment:
+      HOMEBREW_NO_AUTO_UPDATE: "1"
+    steps:
+      - checkout
+      - run: 
+          name: Install Cocoapods
+          command: |
+            brew install cocoapods
+      - run:
+          name: Update Pod repo
+          command: |
+            pod repo update --silent
```

Comment:
We don't have any dependencies, why is this needed?

Reply:
Noticed this as well. I migrated the travis.yml in the old PR which included this command but I agree, we don't have dependencies, so we don't need it.